### ISSUE TITLE
The comman "operator-sdk olm-catalog pogen-csv --csv-version <version>" is not populating the csv with "customresourcedefinitions""

### ISSUE THREAD

**Issue by camilamacedo86** [operator-framework/operator-sdk#1832]
## Bug Report

Bug

**What did you do?**
- Following: https://github.com/operator-framework/operator-sdk/blob/master/doc/user/olm-catalog/generating-a-csv.md#first-generation
- Run `operator-sdk olm-catalog gen-csv --csv-version 0.0.1`

**What did you expect to see?**
The csv be populated with `spec.customresourcedefinitions.owned`

**What did you see instead? Under which circumstances?**
It was not 

**Environment**
* operator-sdk version:

* go version:

* Kubernetes version information:

* Kubernetes cluster kind: 

* Are you writing your operator in ansible, helm, or go?

**Possible Solution**

**Additional context**

**Comment by camilamacedo86** [comment]
Hi folks 

According to the docs shows my understand is that the SDK should already have it implemented. However, when I am looking at the code in the master branch shows that it was not implemented at all. Also, I checked that we have a doc as a proposal for that: https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/sdk-code-annotations.md. 

How do we know what is done and what is done when we are looking these docs? Are all that is implemented removed?

**Comment by estroz** [comment]
@camilamacedo86 you are correct, the CSV generation docs do say that owned CRDs will be [added](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/olm-catalog/generating-a-csv.md#first-generation) to a CSV. That section should have been added in #1162, which implements that feature. So this isn't necessarily a bug. However, #1132 already describes this issue. I will make a PR that updates the CSV generation doc and proposal, and add the current owned CRDs section to #1162.

**Comment by camilamacedo86** [comment]
HI @estroz, 

Really thank you for your reply and info provided.

**Comment by camilamacedo86** [comment]
Hi @estroz,

I could use the PR which is already against master in a project with many specs and all shows worked fine. So, when you merge the PR it will be solved. In this way, should we not close this issue? Have we not another one to track the PR needs. WDYT?
